CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
```

Comment:
@endamccormack can you remove all comments except lines 4 and 5? Everything else sounds like user manual that can be found online. Feel free to link to the https://help.github.com/articles/about-codeowners if you want.